mava/systems/ippo/networks.py

```diff
@@ -82,7 +83,7 @@ def forward_fn(
             rng_key: networks_lib.PRNGKey,
             mask: chex.Array = None,
             policy_state: Tuple[jnp.ndarray] = None,
-        ) -> Tuple[Any, Any, Any]:
+        ) -> Tuple[NestedArray, NestedArray, NestedArray]:
```

Comment:
@DriesSmit, mava has its own NestedArray type. https://github.com/instadeepai/Mava/blob/develop/mava/types.py

As I said, I think we should try not to rely on Acme for things like typing.

Reply:
O okay. I thought you wanted to use Acme's typing. My bad. Will fix it now.

Reply:
But Mava's NestedArray is Any? Is this fine?

Reply:
Yes, acme's is as well :) The typing here is more for readability.

Reply:
I updated all the NestedArray's in Mava to point to Mava's implementation :)